Sources/SpeziAccountPhoneNumbers/PhoneNumbersKey.swift

```diff
@@ -0,0 +1,72 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import SpeziAccount
+import SpeziFoundation
+import SpeziViews
+import SwiftUI
+
+
+private struct DisplayView: SetupDisplayView {
+    typealias Value = [String]
+    @Environment(Account.self)
+    private var account
+    @State var phoneNumberViewModel = PhoneNumberViewModel()
+
+    private var phoneNumbers: Value
+    
+    
+    var body: some View {
+        Section {
+            NavigationLink {
+                PhoneNumbersDetailView(phoneNumbers: account.details?.phoneNumbers ?? [], phoneNumberViewModel: $phoneNumberViewModel)
+            } label: {
+                HStack {
+                    Text("Phone Numbers")
+                    Spacer()
+                    Group {
+                        if let phoneNumber = phoneNumbers.first, phoneNumbers.count == 1 {
+                            Text(phoneNumberViewModel.formatPhoneNumberForDisplay(phoneNumber))
+                        } else if phoneNumbers.count > 1 {
+                            Text("\(phoneNumbers.count) numbers")
+                        } else {
+                            EmptyView()
+                        }
+                    }
+                    .foregroundColor(.secondary)
```

Comment:
SwiftUI offers `LabeledContent` to do exactly this. Spezi offers `ListRow` to add some accessibility modifiers that suits a labeled content view appearing as a list row. This makes sure this view looks great even with large dynamic text sizes or on large screen sizes.

```suggestion
                ListRow("Phone Numbers") {
                    if let phoneNumber = phoneNumbers.first, phoneNumbers.count == 1 {
                        Text(phoneNumberViewModel.formatPhoneNumberForDisplay(phoneNumber))
                    } else if phoneNumbers.count > 1 {
                        Text("\(phoneNumbers.count) numbers")
                    } else {
                        EmptyView()
                    }
                }
```